spyder/plugins/editor/widgets/codeeditor/multicursor_mixin.py

```diff
@@ -538,9 +539,15 @@ def multi_cursor_paste(self, clip_text):
         self.skip_rstrip = True
         self.sig_will_paste_text.emit(clip_text)
         lines = clip_text.splitlines()
-
-        if len(lines) == 1:
-            lines = itertools.repeat(lines[0])
+        
+        if CONF.get('editor', 'multicursor_paste/always_full'):
```

Comment:
```suggestion
        if self.get_conf('multicursor_paste/always_full'):
```

* `CodeEditor` inherits from `TextEditBaseWidget`, which in turn inherits from `SpyderWidgetMixin`, and that class includes this method.
* This is the current way to access config options from any widget (instead of using the `CONF` singleton directly).
* I missed to tell you that in PR #23463, sorry (I'll fix the `CONF` usage in `MouseShortcutEditor` in a follow-up PR).